src/mobile-token/utils.ts

```diff
@@ -22,5 +23,13 @@ export const hasNoTokenType = (remoteToken?: RemoteToken) =>
 export const getDeviceName = (remoteToken?: RemoteToken) =>
   remoteToken?.keyValues?.find((kv) => kv.key === 'deviceName')?.value;
 
+export const getDeviceNameForCurrentToken = (
+  activatedToken?: ActivatedToken,
+  remoteTokens?: RemoteToken[],
+) =>
```

Comment:
Can contemplate changing this to take in `tokenId` as param instead of `ActivatedToken`, as that would make it more reusable.